### PR TITLE
Gaj/feature/subscriptions format

### LINKED ISSUE #12: FE Integration Testing w Cypress
As a dev, I should be able to apply integration testing via Cypress to test. 

Will require reading the documentation first for Redux.

### PR DESCRIPTION
**Pull request by gaj23**
### Description

This adds Cypress testing for `Subscriptions`. 

#### Closes Issues NA

Please see the `cypress` folder inside of `web`. Please see tests in `Subscriptions_spec.js`
  
## Screenshots if applicable (Before/After)

Cypress tests might need to be revisited once backend/real data is applied & changes are updated.

### CODE REVIEW

File: code/web/cypress/integration/Subscription_spec.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        .type('user@crate.com')
+      .get('input[type=password]')
+        .type('123456')
+      .get('button').contains('Login').click()
+  })
+
+  it('Should be able to subscribe to a new crate', () => {
+    cy.get('.jsx-511674265').first()
+      .get('button').first().click()
+      .location().should(loc => {
```
Comment (tashiad) [inline code/web/cypress/integration/Subscription_spec.js:22]: I haven't used this one before! Cool.

File: code/web/src/modules/user/Subscriptions.js [hunk 2]
```diff
@@ -25,7 +28,8 @@ class Subscriptions extends PureComponent {
 
   // Runs on client only
   componentDidMount() {
-    this.props.getListByUser()
+    this.
+    props.getListByUser()
```
Comment (tashiad) [inline code/web/src/modules/user/Subscriptions.js]: any reason why this is on the next line?

**Review by tashiad — APPROVED** [review]

File: code/web/cypress/integration/Subscription_spec.js (hunk 1 above)
Comment (gaj23) [reply code/web/cypress/integration/Subscription_spec.js:22]: The .location is super helpful! It's a nice confirmation that the url and the ux match.

File: code/web/src/modules/user/Subscriptions.js (hunk 2 above)
Comment (gaj23) [reply code/web/src/modules/user/Subscriptions.js]: Nope! Nice catch!

File: code/web/cypress/integration/Subscription_spec.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        expect(loc.pathname).to.eq('/user/subscriptions')
+    })
+    cy.get('.jsx-230203545 div').should('be.visible')
```
Comment (srslie) [inline code/web/cypress/integration/Subscription_spec.js:25]: Glad you and Rachel got these auto-populated div names figured out.

**Review by srslie — APPROVED** [review]
Looks great, thanks Gabby!

**Pull request merged**